packages/tooltips/src/styled/StyledLightTooltip.ts

```diff
@@ -5,27 +5,62 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import classNames from 'classnames';
-import TooltipStyles from '@zendeskgarden/css-tooltips';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import styled, { DefaultTheme, css, ThemeProps } from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import stripUnit from 'polished/lib/helpers/stripUnit';
 
 import { StyledTooltip, IStyledTooltipProps } from './StyledTooltip';
+import { StyledTitle } from './StyledTitle';
 
 const COMPONENT_ID = 'tooltip.light_tooltip';
 
+const arrowStyles = ({ theme, size }: IStyledTooltipProps & ThemeProps<DefaultTheme>) => {
+  let arrowFontSize = `${theme.space.base * 1.75 + 2}px`;
+
+  if (size === 'large') {
+    arrowFontSize = `${stripUnit(theme.fontSizes.sm) + 2}px`;
+  } else if (size === 'extra-large') {
+    arrowFontSize = `${theme.space.base * 4 + 2}px`;
+  }
+
+  return css`
+    &::before,
+    &::after {
+      font-size: ${arrowFontSize};
+    }
+  `;
+};
+
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  return css`
+    border: ${theme.borderWidths.sm} ${theme.borderStyles.solid}
+      ${getColor('neutralHue', 300, theme)};
+    box-shadow: 0 ${theme.space.base * 3}px ${theme.space.base * 5}px 0
```

Comment:
Use the `shadows.lg` theming function here. Improve code readability by assigning a `boxShadow` variable outside of the `css` block.